{{cookiecutter.project_slug}}/tests/unit/test_package.py

```diff
@@ -14,6 +14,9 @@
 
 @pytest.fixture(scope='session')
 def pylintrc(git_root_dir: Path):
+    if not git_root_dir:
```

Comment:
the idea is to find the .pylintrc in the git repo instead of multiplicating them in each service. maybe I can use a backup .pylintrc if it is not available?